vita3k/gui/src/settings.cpp

```diff
@@ -590,6 +588,9 @@ static const std::vector<std::string> LIST_SYS_LANG = {
     "Dansk", "Norsk", "Polskis", u8"Português (Brasil)", "English (United Kingdom)", u8"Türkçe"
 };
 
+static const ImU32 ARROW_COLOR = 4294967295; // White
+static float set_scroll_pos, current_scroll_pos, max_scroll_pos;
```

Comment:
These variables are better, to be moved in GuiState

Reply:
No, is variable for settings only keep here

Reply:
current_scroll_pos is definitely used again from previous state

Reply:
No understund is full fine here